src/events/interactionCreate.ts

```diff
@@ -25,6 +28,26 @@ export default async function (client: CustomClient, interaction: Interaction) {
   //   }
   // }
 
+  if (interaction.isModalSubmit()) {
+    const goodPerson = interaction.fields.getTextInputValue("goodPersonInput");
+    const soWhy = interaction.fields.getTextInputValue("soWhyInput");
+    await interaction.reply("Application sent successfully.");
+    const username = interaction.user.username;
+    const discriminator = interaction.user.discriminator;
+    const authorId = interaction.user.id;
+    const embed = new EmbedBuilder()
+      .setColor(0xaa00aa)
+      .setTitle(`Application of ${username}#${discriminator} (${authorId})`)
+      .addFields(
+        { name: "Are you a good person?", value: goodPerson },
+        { name: "So why do you want to be staff?", value: soWhy }
+      );
```

Comment:
Instead of hardcoding these names here, pull them from an array of {fieldCustomId: string, fieldText: string, fieldStyle: FieldStyle} in the staffapp command, and iterate. Makes it easier to expand and/or change in the future.